src/w3d/lib/hash.cpp

```diff
@@ -34,7 +34,7 @@ HashTableClass::HashTableClass(int size)
 HashTableClass::~HashTableClass()
 {
     if (m_hashTable) {
```

Comment:
While at it, can also write `m_hashTable != nullptr` for style.

Reply:
Well, to be honest the if check makes no sense - delete on nullptr is valid